bin/travis-test.sh

```diff
@@ -3,4 +3,4 @@ set -ev
 
 LERNA=$TRAVIS_BUILD_DIR/node_modules/.bin/lerna
 
-$LERNA run test:ci
+npm run test:ci
```

Comment:
I'd rather leave this as `$LERNA` - we might want to parallelise this at some point, which Lerna lets us do

Reply:
Does that not work when `lerna` is run by `npm`? I changed this so that `nyc` doesn't throw a 'not found' error when run from the command line during local testing but it's probably not needed in practice.

I can change it to `nyc lerna run test:ci` here.

Reply:
other way around - lerna runs npm, and can do so in parallel with the `--parallel` flag. if there's a good reason for changing it though, i don't have a problem, if we find out the tests are too slow we can fix it then.

Reply:
I'll add `--parallel` to the script in `package.json` and see what happens.

Usually, you'd generate coverage reports by running each test script through `nyc`, but [coveralls doesn't support multiple reports for a single repo](https://github.com/lemurheavy/coveralls-public/issues/1174), so I'm running all the tests with `nyc` at the top level.

Reply:
By the way, with the setup here, I'm still seeing lerna run npm in each of the package directories, so I _think_ we're ok.

Reply:
`nyc lerna run --parallel test:ci` works for me locally, so I've added it to this PR.

Reply:
👍